src/meta_protocol_proxy/filters/router/router_impl.cc

```diff
@@ -81,8 +67,46 @@ FilterStatus Router::onMessageDecoded(MetadataSharedPtr request_metadata,
   auto metadata_clone = request_metadata_->clone();
 
   route_entry_->requestMutation(request_mutation);
-  upstream_request_ =
-      std::make_unique<UpstreamRequest>(*this, conn_pool_data, request_metadata_, request_mutation);
+
+  if (decoder_filter_callbacks_->multiplexing()) {
+    // if multiplexing, send by upstream handler
+    auto upstream_handler = decoder_filter_callbacks_->getUpstreamHandler(cluster_name, *this);
+    if (upstream_handler == nullptr) {
+      emitLogEntry(request_metadata_, nullptr, static_cast<int>(ResponseStatus::Error),
+                   "no_matched_upstream_handler");
```

Comment:
Can we put a more precise error description, for examle: "unknow cluster ....", "no conn pool for {}", in the log entry instead of "no_matchec_upstream_hander" ?